src/test/kotlin/com/figure/gradle/semver/BuildLogicFunctionalSpec.kt

```diff
@@ -68,17 +68,14 @@ class BuildLogicFunctionalSpec : FunSpec({
         ) { task: String ->
             // first one loads the cache
             val firstRun = runner
-                .withArguments(task, "--configuration-cache", "--scan")
+                .withArguments(task, "--configuration-cache")
                 .build()
 
-            print(firstRun.output)
-
             // second one uses the cache
             val secondRun = runner
-                .withArguments(task, "--configuration-cache", "--scan")
+                .withArguments(task, "--configuration-cache")
                 .build()
 
-            firstRun.output shouldContain "0 problems were found storing the configuration cache."
```

Comment:
It seems some of the logging that Gradle does has changed, so our assertions here have been tweaked.